CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ endif ()
 message(STATUS "CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}")
 
 cmake_minimum_required(VERSION 3.14)
-project(MMDeploy VERSION 0.13.0)
+project(MMDeploy VERSION 1.1.0)
```

Comment:
SDK skips `1.0.0`?

Reply:
maybe it should be synced with `mmdeploy_runtime` since `mmdeploy_runtime` is `v1.1.0`

Reply:
I am still struggling to convince myself